tests/unit/test_main.py

```diff
@@ -239,7 +239,7 @@ def event_loop_mock(mocker, monkeypatch):
 @pytest.mark.parametrize('has_active_plugins,exp_log_count,errors', (
     (True, 2, []),
     (True, 2, [('not_a.plugin', conftest.plugin_exc_mock())]),
-    (False, 2, []),
+    (False, 1, []),
```

Comment:
I think this originally passed because of an error message before this was fixed.